bfvmm/src/vmcs/test/test_vmcs_intel_x64_check_controls.cpp

```diff
@@ -0,0 +1,1089 @@
+//
+// Bareflank Hypervisor
+//
+// Copyright (C) 2015 Assured Information Security, Inc.
+// Author: Rian Quinn        <quinnr@ainfosec.com>
+// Author: Brendan Kerrigan  <kerriganb@ainfosec.com>
+// Author: Connor Davis      <davisc@ainfosec.com>
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+
+#include <test.h>
+#include <vmcs/vmcs_intel_x64.h>
+#include <memory_manager/memory_manager.h>
+#include <vector>
+#include <functional>
+
+enum path_type { TEST_FAILURE = 0, TEST_SUCCESS = 1 };
+
+extern std::map<uint32_t, uint64_t> g_msrs;
+extern std::map<uint64_t, uint64_t> g_vmcs_fields;
+extern bool g_phys_to_virt_return_nullptr;
+
+bool vmread(uint64_t field, uint64_t *val);
+uint64_t read_msr(uint32_t msr);
+uint32_t cpuid_eax(uint32_t val);
+void *physint_to_virtptr(uintptr_t phys);
+
+#define RUN_VMCS_TEST(func) \
+    do \
```

Comment:
Please convert this to a function. You could either use a template function, or if the signature is known, you could define the signature up front. That should clear up the Astyle issue as well. If it doesn't we can add blockers to Astyle as a last ditch effort.